### PR TITLE
Better rubocop

### PR DESCRIPTION
**Pull request by cbeer**
Depends on #17

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+# Turn on RSpec cops
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.2
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
```
Comment (mjgiarlo) [inline .rubocop.yml]: Do we want to ignore `db` and `config` entirely?

**Comment by mjgiarlo** [comment]
One rubocop failure w/r/t `spec/spec_helper.rb`, so currently :red_circle: 

I am fine with these changes once the build is green. :+1:

File: .rubocop.yml (hunk 1 above)
Comment (cbeer) [inline .rubocop.yml]: `db`, I'd argue certainly. Migrations are the only user-maintained thing in there, and most of them are from upstream.

`config`, I could do either way. :shrug:

Comment (mjgiarlo) [inline .rubocop.yml]: OK, :+1: to leaving `db` and :-1: to `config`, then.

**Pull request merged**